custom_components/unifi_insights/api/network/models/device.py

```
@@ -179,3 +179,147 @@ class LegacyPortMetrics(BaseModel):
+    """
+    if not isinstance(legacy_device, dict):
+        return LegacyOutletMetrics()
+
+    raw_table = legacy_device.get("outlet_table")
+    if not isinstance(raw_table, list):
+        raw_table = []
+
+    def _to_float(value: Any) -> float | None:
+        try:
+            return float(value)
+        except TypeError, ValueError:
```

Pylint (4.0.7)

[error] Parsing failed: 'multiple exception types must be parenthesized (unifi_insights.api.network.models.device, line 231)'

(E0001)